flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -527,23 +527,41 @@ public static String[] mergeListsToArray(List<String> base, List<String> append)
      * open. Unlimited be default.
      */
     public static ConfigOption<Integer> fileSystemConnectionLimit(String scheme) {
-        return ConfigOptions.key("fs." + scheme + ".limit.total").intType().defaultValue(-1);
+        return ConfigOptions.key("fs." + scheme + ".limit.total")
+                .intType()
+                .defaultValue(-1)
+                .withDescription(
+                        "The connection limit for the file system "
+                                + scheme
+                                + ". The valid value must >= -1. Values of -1 or 0 will be ignored.");
```

Comment:
nit: will be ignored -> means there is no connection limit

Reply:
Done.